### PR TITLE
Add Terms of Service for SparkRewards

### PR DESCRIPTION
**Pull request by mativanarquero**

### CODE REVIEW

File: sparkrewards/index.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    <p><strong>Changes to This Terms and Conditions</strong></p>
+    <p>
+      We may update our Terms and Conditions
+      from time to time. Thus, you are advised to review this page
+      periodically for any changes. We will
+      notify you of any changes by posting the new Terms and
+      Conditions on this page.
+    </p>
+    <p>
+      These terms and conditions are effective as of 2021-04-02
```
Comment (harveyjavier) [inline sparkrewards/index.html:94]: Modify here the date here as well with the initial roll out of the app

**Review by harveyjavier — CHANGES_REQUESTED** [review]

**Review by harveyjavier — APPROVED** [review]

File: sparkrewards/index.html (hunk 1 above)
Comment (mativanarquero) [reply sparkrewards/index.html]: [Updated](https://github.com/sparkpointio/terms_and_conditions/pull/7/commits/dded674f5e63a7bb8502f51c298b17e8e6311611) @harveyjavier for checking thank you

**Comment by harveyjavier** [comment]
na draft mu ata Mat. may imomodify pa u?

**Comment by mativanarquero** [comment]
> na draft mu ata Mat. may imomodify pa u?

Oh sorry, ready for review na

**Pull request merged**